test/commonModal.test.js

```diff
@@ -0,0 +1,40 @@
+import { getByTestId } from '@testing-library/dom';
+import '@testing-library/jest-dom';
+import {
+  hideElement,
+  renderComponentIntoSelectedTagID,
+} from '../src/utils/functions';
+import { CommonModal } from '../src/components/commonModal/CommonModal';
+import userEvent from '@testing-library/user-event';
+
+describe('Test common modal', () => {
+  beforeAll(() => {
+    document.body.innerHTML = `<div id="swquiz-app">
```

Comment:
```suggestion
    document.body.innerHTML = mockedDomStructure
```
Templatkę moglibyśmy sobie wyrzucić do osobnego pliku, przyda się każdemu do testów.